backend/src/services/v2/http.ts

```diff
@@ -0,0 +1,7 @@
+import https from 'node:https'
+
+// This function has the same syntax as 'https.Agent', but is centralised throughout
+// the application in case it needs to be altered.
+export function getHttpsAgent(config?: https.AgentOptions) {
+  return new https.Agent({ ...config })
```

Comment:
Do we want to create a new agent on every call to this function or should we be reusing a single agent?

Could do something like:
```
// Not sure what the correct type is off the top of my head
let httpsAgent: HttpsAgent | undefined

export function getHttpsAgent(config?: https.AgentOptions) {
  if (!httpsAgent) {
    httpsAgent = new https.Agent({ ...config })
  }
  return httpsAgent
}
```